README.md

```diff
@@ -1,16 +1,18 @@
+# picknik_controllers
 
+Package defining custom controllers that are generic and we want to contribute them into ros2_controllers.
+The controllers have been prefixed with `picknik_` so that when they eventually land upstream the prefix can be removed and the versions here will become deprecated.
 
-# kortex2_controllers
-Package defining custom controllers created for kinova robots.
-- `kortex2_controllers::TwistController`
-- `kortex2_controllers::FaultController`
+- `picknik_twist_controller::PicknikTwistController`
+- `picknik_reset_fault_controller::PicknikResetFaultController`
 
 
-## TwistController
-Cartesian twist controller created to use builtin controller on the kinova robot.
+## PicknikTwistController
+Cartesian twist controller created to use builtin controller on the kinova robot but generic and can work with other robots.
```

Comment:
I think we can remove the mention of its use in the Kinova drivers. How does the following sound?
```suggestion
Generic ROS2 controller to forward cartesian twist commands. 
```